### PR TITLE
Make cache detection thersholds configurable on the MediaPlayer API

### PR DESCRIPTION
**Pull request by AxelDelmas**

### CODE REVIEW

File: src/streaming/MediaPlayer.js [hunk 1]
```diff
@@ -1565,6 +1565,78 @@ function MediaPlayer() {
     }
 
     /**
+     * Latency threshold in ms under which we that a segment request came back from cache
```
Comment (ruslandinov) [inline src/streaming/MediaPlayer.js]: I think there is `assume` or `think` missing between `we` and `that`.

File: src/streaming/MediaPlayer.js [hunk 2]
```diff
@@ -1565,6 +1565,78 @@ function MediaPlayer() {
...
+     * @see {@link module:MediaPlayer#setCacheLoadThresholdLatency setCacheLoadThresholdLatency()}
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function getCacheLoadThresholdLatency() {
+        return mediaPlayerModel.getCacheLoadThresholdLatency();
+    }
+
+    /**
+     * Download time threshold in ms under which we that a video segment request came back from cache
```
Comment (ruslandinov) [inline src/streaming/MediaPlayer.js]: Same here

File: src/streaming/MediaPlayer.js [hunk 3]
```diff
@@ -1565,6 +1565,78 @@ function MediaPlayer() {
...
+     * @see {@link module:MediaPlayer#setCacheLoadThresholdVideo setCacheLoadThresholdVideo()}
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function getCacheLoadThresholdVideo() {
+        return mediaPlayerModel.getCacheLoadThresholdVideo();
+    }
+
+    /**
+     * Download time threshold in ms under which we that an audio segment request came back from cache
```
Comment (ruslandinov) [inline src/streaming/MediaPlayer.js]: idem

**Review by ruslandinov — APPROVED** [review]
👍

**Comment by AxelDelmas** [comment]
Thanks @ruslandinov

**Pull request merged**